### DISCUSSION TITLE
Display detailed values for all items in DifficultyAdjust

### DISCUSSION THREAD

**Discussion by shiumano** [ppy/osu#27740] [Ideas]
Now AR details value (ms) is displayed when expanding the limit
Why not display the values for other items as well?
example
CS: Magnification compared to the size with a CS value of 0, or the ratio to the play field
HP: Time until HP reaches 0, or amount of HP reduction per second
OD: Range of time when GREAT (300s) judgment occurs

**Comment by Rekunan** [comment]
It's also good cause it maintains parity with stable:
![image](https://github.com/ppy/osu/assets/24501632/f395b73b-7b8b-408c-929a-ca4ff8347b35)

Although this also means we're free to decide what to show for HP. According to the ~~incomplete~~[wiki page](https://osu.ppy.sh/wiki/en/Beatmap/HP_drain_rate), there is HP drain rate, miss penalty, and hit reward.

Looking through the code itself, starting from [HealthProcessor](https://github.com/ppy/osu/blob/master/osu.Game/Rulesets/Scoring/HealthProcessor.cs) to [Judgement](https://github.com/ppy/osu/blob/master/osu.Game/Rulesets/Judgements/Judgement.cs), it seems like the HP hit reward isn't affected by the HP value, only the judgement, so that wouldn't be helpful.

But I am now reminded that there's a classic health option on CL, since there's a [DrainingHealthProcessor](https://github.com/ppy/osu/blob/master/osu.Game/Rulesets/Scoring/DrainingHealthProcessor.cs) and a [LegacyDrainingHealthProcessor](https://github.com/ppy/osu/blob/master/osu.Game/Rulesets/Scoring/LegacyDrainingHealthProcessor.cs), which makes me think it's better to not show anything for HP to save on complexity. I'm not even sure if there's a simple `DrainRate` value in the Legacy processor, since it seems like `testDrop` just changes depending on the health itself, and not the HP value.

**Reply by shiumano** [reply]
I found something that looks like it.
#6202
